kartothek/io/dask/bag.py

```diff
@@ -76,6 +76,15 @@ def read_dataset_as_metapartitions_bag(
         load_dataset_metadata=load_dataset_metadata,
     )
 
+    if len(ds_factory.tables) > 1:
+        warnings.warn(
+            "Trying to read a dataset with multiple internal tables. This functionality will be removed in the next "
+            "major release. If you require a multi tabled data format, we recommend to switch to the kartothek Cube "
+            "functionality. "
+            "https://kartothek.readthedocs.io/en/stable/guide/cube/cube_design_features.html?highlight=multi%20table#multi-dataset-vs-multi-table",
```

Comment:
Please just point to https://kartothek.readthedocs.io/en/stable/guide/cube/kartothek_cubes.html

Reply:
Done.